internal/view/live_view.go

```diff
@@ -286,16 +287,20 @@ func (v *LiveView) toggleFullScreenCmd(evt *tcell.EventKey) *tcell.EventKey {
 		return evt
 	}
 
-	v.fullScreen = !v.fullScreen
-	v.SetFullScreen(v.fullScreen)
-	v.Box.SetBorder(!v.fullScreen)
-	if v.fullScreen {
+	v.setFullScreen(!v.fullScreen)
+
+	return nil
+}
+
+func (v *LiveView) setFullScreen(isFullScreen bool) {
```

Comment:
Likely we should refactor Live/Details view to offer this functionality.

Reply:
I added same functionality for Details view, if you could kindly check if thats what you meant by "refactor Live/Details view" that would be great.

Reply:
Right I was thinking combining the 2 to dry this up in a single place. This feels like a sep PR tho.